src/main.rs

```diff
@@ -291,12 +288,12 @@ struct CheckRelease {
     verbosity: clap_verbosity_flag::Verbosity<clap_verbosity_flag::InfoLevel>,
 
     /// Whether to print colors to the terminal:
-    /// always, alwaysansi (always use only ANSI color codes),
+    /// always, always-ansi (always use only ANSI color codes),
     /// auto (based on whether output is a tty), and never
     ///
     /// Default is auto (use colors if output is a TTY, otherwise don't use colors)
-    #[arg(value_enum, long = "color")]
-    color_choice: Option<termcolor::ColorChoice>,
+    #[command(flatten)]
+    color_choice: colorchoice_clap::Color,
```

Comment:
I might be confused here, but `colorchoice_clap::Color` seems like it might not have `alwaysansi` as an option, whereas `anstream::ColorChoice` does. Do we want to use `anstream::ColorChoice` here?

Reply:
IIRC, `AlwaysAnsi` is skipped in `clap` because `Always` always resolves to it except when running on a Windows console. And when someone's using a Windows console, they wouldn't want to use `AlwaysAnsi` option anyway.

Code for reference: https://github.com/rust-cli/anstyle/blob/ed1a598cc2c3b84483c6058a982d045f7f7aecfc/crates/anstream/src/auto.rs#L110-L126